pkgs/development/python-modules/sphinx-hoverxref/default.nix

```
@@ -20,7 +20,7 @@
 
 buildPythonPackage rec {
   pname = "sphinx-hoverxref";
-  version = "1.3.0";
+  version = "1.5.0";
   format = "pyproject";
```

```suggestion
  pyproject = true;
```
and use `build-system`, `dependencies`